### ISSUE TITLE
Allow empty string to be rendered

### ISSUE THREAD

**Issue by serv** [gdub22/ember-cli-inline-content#15]
In this example (https://github.com/gdub22/ember-cli-inline-content#rendering-a-string-of-content-instead-of-a-file), when you attempt to add empty string `''` as content, you get this error.

```
Path must be a string. Received { content: '' }
TypeError: Path must be a string. Received { content: '' }
    at assertPath (path.js:7:11)
    at Object.join (path.js:1211:7)
    at InlineContentRenderer.readFile (/Users/jasonkim/work/node/webclient-cli/node_modules/ember-cli-inline-content/index.js:68:23)
    at InlineContentRenderer.contentFor (/Users/jasonkim/work/node/webclient-cli/node_modules/ember-cli-inline-content/index.js:43:22)
    at /Users/jasonkim/work/node/webclient-cli/node_modules/ember-cli/lib/broccoli/ember-app.js:1539:49
    at Array.reduce (native)
    at EmberApp.contentFor (/Users/jasonkim/work/node/webclient-cli/node_modules/ember-cli/lib/broccoli/ember-app.js:1538:33)
    at replacement (/Users/jasonkim/work/node/webclient-cli/node_modules/broccoli-config-replace/index.js:86:36)
    at RegExp.[Symbol.replace] (native)
    at RegExp.[Symbol.replace] (native)
```

I think this is intended. We should be able to render empty string if we want to.

**Comment by gdub22** [comment]
Yep. Check undefined instead of falsey here: https://github.com/gdub22/ember-cli-inline-content/blob/master/index.js#L39

**Comment by serv** [comment]
When I said `I think this is intended.` earlier, I meant to say `I think this is unintended.`

Do you think it is okay the way it's currently working now or do you agree that we should allow empty string?

**Comment by gpoitch** [comment]
I don't know why you would want to do that, but I think it was a bug anyway. Fixed and released 0.4.1
